views/layout.njk

```diff
@@ -2,6 +2,9 @@
 <html style="background-color: #a6549d;">
   <head>
     <title>{% block title %}immers.space{% endblock %}</title>
+    {% if monetizationPointer %}
+    <meta name="monetization" content="{{ monetizationPointer }}">
+    {% endif %}
```

Comment:
@quinn-madson I double-checked based on our convo and this does indeed avoid writing the monetization tag if a pointer is not provided, so I think this PR is g2g as well